### PR TITLE
feat: Filter python tests

### PR DESCRIPTION
**Pull request by dstenroejl**
We want to split python tests so we can run some on one GitHub runner and others on another runner, so they run in parallel and our CI pipeline becomes faster.

To do this we need to be able to define a filter that pytest can understand.

### CODE REVIEW

**Review by BjarkeMeier — APPROVED** [review]

File: .github/workflows/ci-databricks.yml [hunk 1]
```diff
@@ -105,20 +105,31 @@ jobs:
...
       azure_integrationtest_spn_id: ${{ vars.integration_test_azure_spn_id_oidc }}
       azure_keyvault_url: ${{ vars.integration_test_azure_keyvault_url }}
 
+  python_tests_other:
+    uses: Energinet-DataHub/.github/.github/workflows/python-ci.yml@v12
+    with:
+      operating_system: dh3-ubuntu-20.04-4core
+      path_static_checks: ./source/databricks/calculation_engine
+      # documented here: https://github.com/Energinet-DataHub/opengeh-wholesale/tree/main/source/databricks#styling-and-formatting
+      ignore_errors_and_warning_flake8: E501,F401,E402,E203,W503
```
Comment (djorgensendk) [inline .github/workflows/ci-databricks.yml:129]: I don't know if the error and warning to ignore changes regularly, but perhaps you could consider making that range the default value of the parameter

File: .github/workflows/ci-databricks.yml [hunk 2]
```diff
@@ -105,20 +105,31 @@ jobs:
...
       use_integrationtest_environment: true
       azure_integrationtest_tenant_id: ${{ vars.integration_test_azure_tenant_id }}
       azure_integrationtest_subscription_id: ${{ vars.integration_test_azure_subscription_id }}
       azure_integrationtest_spn_id: ${{ vars.integration_test_azure_spn_id_oidc }}
       azure_keyvault_url: ${{ vars.integration_test_azure_keyvault_url }}
 
+  python_tests_other:
+    uses: Energinet-DataHub/.github/.github/workflows/python-ci.yml@v12
+    with:
+      operating_system: dh3-ubuntu-20.04-4core
```
Comment (djorgensendk) [inline .github/workflows/ci-databricks.yml:126]: same as the other comment - possible default value?

**Review by djorgensendk — APPROVED** [review]

File: .github/workflows/ci-databricks.yml (hunk 2 above)
Comment (dstenroejl) [reply .github/workflows/ci-databricks.yml:126]: Here we use a large runner. Actually this is probably not even a good runner configuration. We experimented with this maybe a year ago and found this level to be more efficient when executing python tests in parallel on the same runner. We should conduct a similar investigation to find the best runner configuration matching todays usage pattern. But it's another user story.

File: .github/workflows/ci-databricks.yml (hunk 1 above)
Comment (dstenroejl) [reply .github/workflows/ci-databricks.yml:129]: It's a good idea, but I would like to postpone it to when we (maybe in cooperation with the platform team), clean up the workflows for python.

**Pull request merged**